azure/opsverse-aks/backend.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "<= 4.4.0"
+    }
+  }
+
+  backend "azurerm" {
+    resource_group_name  = "<->"         # Resource group of pre-created storage account for storing Terraform state.
```

Comment:
Also, I am guessing the `resouce_group` should already exist before a customer starts cluster creation to store terraform state remotely. Have we added this note/info in the README? If no, please add.

Reply:
@amar-opsverse ^